omnipaxos/tests/config/test.toml

```diff
@@ -64,3 +64,9 @@ storage_type = { type = "Broken" }
 [sync_test]
 wait_timeout_ms = 5000
 num_nodes = 3
+
+[ui_test]
+wait_timeout_ms = 5000
+num_nodes = 3
+num_threads = 8
+election_timeout_m = 100
```

Comment:
We have too many variants of these test configurations. If you only need something similar to the default values, then use `TestConfig::default()` in the code instead.

https://github.com/haraldng/omnipaxos/blob/master/omnipaxos/tests/utils.rs#L96